jerry-port/default/defaultx-extfunc.c

```diff
@@ -0,0 +1,29 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdio.h>
+
+#include "jerryscript-ext/extfunc.h"
+
+/**
+ * Print a single character to stdout.
+ *
+ * @param c the character to print.
+ */
+void
+jerryx_port_extfunc_print_char (char c) /**< the character to print */
+{
+  printf ("%c", c);
```

Comment:
Note, that without this function, the 'print' handler does not work. (It cannot have `printf` hardwired because we have platforms (targets) where `printf` is unavailable.)

The alternative is to hardwire `printf` into the print handler, but then we will lose reusability, as many targets will have to copy-paste the code, which we wanted to avoid AFAIK.

Having said that, I'm not sure that this is the best approach. And I'm definitely open to suggestions.